.github/workflows/install.yml

```diff
@@ -0,0 +1,52 @@
+name: "Install"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  dist:
+    runs-on: ubuntu-latest
+    name: Python wheel
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        pip install -r requirements-test.txt
+    - name: Build package
+      run: python -m build -o dist/
+    - uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: dist
+
+  test:
+    runs-on: ubuntu-latest
+    needs: [dist]
+    strategy:
+      matrix:
+        python-version: [ '3.7', '3.8', '3.9', '3.10' ]
+        install-from: ["dist/*.whl"]
```

Comment:
We don't run functional tests in a matrix, so I'd say it'd be fine to run on a single version here as well (3.11). This will also reduce the number of jobs.

Reply:
ack